test/Spago.purs

```diff
@@ -37,18 +37,21 @@ main = Aff.launchAff_ $ void $ un Identity $ Spec.Runner.runSpecT testConfig [ S
     -- TODO:
     -- Upgrade set?
     -- Script?
-
+    {-
```

Comment:
Let's run all the tests 😄 

Note that you can use `itOnly` and `describeOnly` to have the `spec` run only that test/suite

Reply:
😅 My bad, thanks for the tip.

Reply:
No worries!